web/admin/application/configs/klear/DestinationRateGroupsList.yaml

```diff
@@ -89,6 +92,9 @@ production:
       class: ui-silk-pencil
       label: false
       title: _("Edit %s %2s", ngettext('Destination rate', 'Destination rates', 1), "[format| (%item%)]")
+      fields:
+        readOnly:
+          deductibleConnectionFee: true
```

Comment:
This should be validated on backend. Entity->sanitize : 
`if (!$this->isNew() && $this->hasChanged('deductibleConnectionFee')) { 
  throw new \DomainException('msg') 
}`